CMakeLists.txt

```diff
@@ -71,6 +71,12 @@ if(SECP256K1_ENABLE_MODULE_EXTRAKEYS)
   add_compile_definitions(ENABLE_MODULE_EXTRAKEYS=1)
 endif()
 
+option(SECP256K1_ENABLE_MODULE_ELLSWIFT "Enable ElligatorSwift module." ON)
+if(SECP256K1_ENABLE_MODULE_ELLSWIFT)
+  add_definitions(-DENABLE_MODULE_ELLSWIFT=1)
```

Comment:
I'm assuming this is older than our `add_compile_definitions` switch-over. Mind updating it to match?